build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -71,8 +71,12 @@ spec:
       containers:
       {{- end }}
         - name: install-cni
-          image: {{ include "antreaImage" . | quote }}
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: {{ include "antreaAgentImage" . | quote }}
+          {{- if .Values.image.pullPolicy }}
+          imagePullPolicy: {{ .Values.image }}
+          {{- else }}
+          imagePullPolicy: {{ .Values.agentImage.pullPolicy }}
+          {{- end }}
```

Comment:
could you define a `antreaAgentImagePullPolicy` helper template for readability?